foxy/distribution.yaml

```diff
@@ -2519,10 +2519,13 @@ repositories:
       url: https://github.com/ros2/rmw_implementation.git
       version: foxy
     release:
+      packages:
+      - rmw_implementation
+      - test_rmw_implementation
```

Comment:
Should this package really be released?

Reply:
It's just a package that contains tests. feel free to close it if this is not relevant or not adding any value.

I was following the steps done in rolling (where is [released]( https://github.com/ros/rosdistro/blob/master/rolling/distribution.yaml#L1629))

Reply:
> I was following the steps done in rolling (where is released)

This was added in #25720. @ivanpauno Can you clarify if that package needs to be made available from Debian or if it only contains tests and should be ignored in the gbp repo instead?

Reply:
I'm not sure what's the procedure to only release some packages in a repository and not all.
I agree that `test_rmw_implementation` don't need to have Debian package.

Reply:
> I'm not sure what's the procedure to only release some packages in a repository and not all.

https://answers.ros.org/question/66545/to-ignore-some-pkgs-upon-release-by-bloom/

@ivanpauno The `.ignore` should be added for rolling and the `test_rmw_implementation` name should be removed from the rolling distribution.
@ahcorde The `test_rmw_implementation` name should be removed from this PR as well. No need to run bloom again - the already generated data for that package will simply not be used.

Reply:
> @ivanpauno The .ignore should be added for rolling

Done in https://github.com/ros2-gbp/rmw_implementation-release/commit/d3cbd4f7ddaf407369733b447e9cbbd42864cbd9#diff-d1bed8395d4099eceeaaba8471aa916b025de36ae35f0993edb489af258f8df7.

Reply:
@ivan Please still remove the package name from the rolling distribution file.

Reply:
> Please still remove the package name from the rolling distribution file.

Ah, I thought this PR was doing that. See https://github.com/ros/rosdistro/pull/27104.

Reply:
Considering this PR, should a `foxy.ignored` file be added too?